index.js

```diff
@@ -12,34 +12,42 @@ console.log(fifaData);
 (d) Away Team goals for 2014 world cup final
 (e) Winner of 2014 world cup final */
 
+console.log(wcf['Home Team Wins']);
+console.log(wcf['Away team Wins']);
+console.log(wcf);
 
 /* Task 2: Create a function called  getFinals that takes `data` as an argument and returns an array of objects with only finals data */
 
-function getFinals(/* code here */) {
-
-    /* code here */
+function getFinals(data) {
+    return data.filter(match => match.Stage ==='Final');
 
 };
+console.log(getFinals(fifaData));
 
 /* Task 3: Implement a higher-order function called `getYears` that accepts the callback function `getFinals`, and returns an array called `years` containing all of the years in the dataset */
 
-function getYears(/* code here */) {
+function getYears(cb) {
 
-    /* code here */
+    return cb(fifaData).map(match => match.Year);
 
 };
 
-getYears();
+console.log(getYears(getFinals));
```

Comment:
Task 3 looks good! You can remove the extra spacings -- developers like to keep their code clean and formatted (: Maybe look into the VSCode extension `Prettier`